include/spdlog/sinks/qt_sinks.h

```diff
@@ -53,50 +62,184 @@ class qt_sink : public base_sink<Mutex>
     std::string meta_method_;
 };
 
+#if !defined(SPDLOG_USE_STD_FORMAT)
+template <typename Mutex>
+class qt_sink_color : public qt_sink<Mutex> {
```

Comment:
I don't see much benefit in inherit from qt_sink here.

Reply:
yes, not too much now I changed to base base_sink